automerge/src/change.rs

```diff
@@ -407,6 +427,13 @@ impl Change {
         self.bytes.compress(self.body_start);
     }
 
+    pub fn compressed_bytes(&self) -> &[u8] {
```

Comment:
This name seems a bit confusing to me if we have the uncompressed variant. Perhaps something just like `bytes`?

Reply:
Done!